core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
+  
+i.oppia-share-btn-fa {
+    display: inline-block;
```

Comment:
@seanlip Sure Sir, will correct them right away. And [ coding style guide ](https://github.com/oppia/oppia/wiki/Coding-style-guide#css) reads .. 

 For directives, include the CSS in the directive template file, similar to what we do in `core/templates/dev/head/components/exploration_summary_tile_directive.html`

But there is **no such file**, which made me unfamiliar with styling the code base.
Looks like the file is moved to  `summary_tile` folder, and right path is `core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html`. 

However, the file you posted in comments give the better idea of the styling guide than former. 
`core/templates/dev/head/pages/library/activity_tiles_infinity_grid_directive.html`

Thank you ! .